src/style/card-categorias-de-imoveis/card-category.css

```
@@ -0,0 +1,14 @@
+.category-card-container {
+  box-shadow: 0px 3px 4px rgba(153, 155, 168, 0.15);
+  cursor: pointer;
+  border-radius: 4px;
+  display: inline-block;
+}
+.img {
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+  overflow: hidden;
+}
+.category-card-text {
+  padding: 0 12px;
```

```suggestion
  padding-horizontal: 12px;
```

Meio nit, porém parece mais descritivo dessa forma pra quem está lendo.

---

existe horizontal no web? Acho que é so no native. 🤔 
@deboraCs26 se nao funcionar o `padding-horizontal`, tenta o `padding-inline`